main.go

```diff
@@ -25,15 +24,25 @@ var (
 	version = "head"
 )
 
+type DockerSecret struct {
+  Value string
+}
+
+func (s *DockerSecret) UnmarshalText(b []byte) error {
+  v, err := os.ReadFile(string(b))
+  s.Value = string(v)
+  return err
+}
+
 type args struct {
 	Addr                 string              `arg:"env:DOZZLE_ADDR" default:":8080" help:"sets host:port to bind for server. This is rarely needed inside a docker container."`
 	Base                 string              `arg:"env:DOZZLE_BASE" default:"/" help:"sets the base for http router."`
 	Level                string              `arg:"env:DOZZLE_LEVEL" default:"info" help:"set Dozzle log level. Use debug for more logging."`
 	TailSize             int                 `arg:"env:DOZZLE_TAILSIZE" default:"300" help:"update the initial tail size when fetching logs."`
-	Username             string              `arg:"env:DOZZLE_USERNAME" help:"sets the username for auth."`
-	Password             string              `arg:"env:DOZZLE_PASSWORD" help:"sets password for auth"`
-	UsernameFILE         string              `arg:"env:DOZZLE_USERNAME_FILE" help:"sets the secret path read username for auth."`
-	PasswordFILE         string              `arg:"env:DOZZLE_PASSWORD_FILE" help:"sets the secret path read password for auth"`
+	Username             *string             `arg:"env:DOZZLE_USERNAME" help:"sets the username for auth."`
```

Comment:
Why does this need to be a pointer now? Keep `string` and not `*string` will be cleaner because you can check for `!= ""`